tests/suites/test_suite_ssl.data

```diff
@@ -10466,6 +10466,22 @@ SSL TLS 1.3 Key schedule: Traffic key generation #2 (RFC 8448)
 depends_on:MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
 ssl_tls1_3_traffic_key_generation:MBEDTLS_MD_SHA256:"a2067265e7f0652a923d5d72ab0467c46132eeb968b6a32d311c805868548814":"ff0e5b965291c608c1e8cd267eefc0afcc5e98a2786373f0db47b04786d72aea":12:16:"844780a7acad9f980fa25c114e43402a":"4c042ddc120a38d1417fc815":"7154f314e6be7dc008df2c832baa1d39":"71abc2cae4c699d47c600268"
 
+SSL TLS 1.3 Key schedule: Derive-Secret( ., "derived", "")
+# Vector from TLS 1.3 Byte by Byte (https://tls13.ulfheim.net/)
+# Derive-Secret( Early-Secret, "derived", "")
+# Tests the case where context isn't yet hashed (empty string here,
+# but still needs to be hashed)
+# 64657269766564 = hex("derived")
+depends_on:MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+ssl_tls1_3_derive_secret:MBEDTLS_MD_SHA256:"33ad0a1c607ec03b09e6cd9893680ce210adf300aa1f2660e1b22e10f170f92a":"64657269766564":"":32:0:"6f2615a108c702c5678f54fc9dbab69716c076189c48250cebeac3576c3611ba"
```

Comment:
I didn't do that initially because `MBEDTLS_SSL_CONTEXT_[UN]HASHED` was guarded by `MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL`, and the test generation script choked on that IIRC. Anyway, I removed those guards in a recent commit (since we don't guard declarations of modules even if they are disabled in the config), so we should be able to use those constants now. I'll change it.